templates/swift/base/params.twig

```diff
@@ -0,0 +1,28 @@
+{% macro methodNeedsSecurityParameters(method) %}
+{% if (method.type == "webAuth" or method.type == "location") and method.security|length > 0 %}{{ true }}{% else %}{{false}}{% endif %}
+{% endmacro %}
+{% if method.parameters.path %}        var{% else %}        let{% endif %} path: String = "{{ method.path }}"
+{% for parameter in method.parameters.path %}
+        path = path.replacingOccurrences(
+          of: "{{ '{' }}{{ parameter.name | caseCamel }}{{ '}' }}",
+          with: {% if method.parameters.path %}{% if parameter.isGlobal %}self.{% endif %}{{ parameter.name | caseCamel | escapeKeyword }}{% else %}""{% endif %}
+        )
+{% endfor %}
+{% if method.parameters.query or method.parameters.body or method.parameters.formData or _self.methodNeedsSecurityParameters(method) %}
+        {% if 'multipart/form-data' in method.consumes %}var{% else %}let{% endif %} params: [String: Any?] = [
+{% else %}
+        let params: [String: Any?] = [:]
+{% endif %}
+{% for parameter in method.parameters.query | merge(method.parameters.body) %}
+            "{{ parameter.name }}": {% if parameter.isGlobal %}self.{% endif %}{{ parameter.name | caseCamel | escapeKeyword }}{% if not loop.last or _self.methodNeedsSecurityParameters(method) %},{% endif %}
```

Comment:
Please add a newline after this one to keep each parameter on their own line

Reply:
made the changes